pset-functions.js

```diff
@@ -91,6 +107,33 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+function wordCount(phrase){
```

Comment:
@carly-ga nice approach, it shows that you put a lot of thought into your plan of attack. Also, the use of `Set` and `Array.from()` shows resourcefulness. Good job getting everything working according to the requirements.

An alternative approach that would result in less complex code would have been a "hash map", where an object is used to construct the word count. 

```
function wordCounter (phrase) {
  const phraseWords = phrase.split(' ')
  const wordMapObj = {} // object that we'll use to keep track of word occurences

  phraseWords.forEach((word) => {
    /* Plan
      1. check of word is already in wordMapObj
       - if word is not in wordMapObj then add it as a key and set value to 1
       - if word is already in wordMapObj then then increment value of word (key) by 1
    */

    if (wordMapObj[word] === undefined) {
      // word does not exist in wordMapObj
      wordMapObj[word] = 1
    } else {
      // when word already exists in wordMapObj
      wordMapObj[word] += 1
    }
  })

  return wordMapObj
}
```